src/Icon/images/blocks.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="46px" height="46px" viewBox="0 0 46 46" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41 (35326) - http://www.bohemiancoding.com/sketch -->
```

Comment:
we should consider running our SVGs through an optimizer before merge to clean up some of the excess size.
things like these comments and the `title`, `desc` and empty elements can be stripped out without affecting the output.